doc/user_guide/dynamo0p3.rst

```diff
@@ -3772,3 +3772,7 @@ all versions of the Dynamo API.
 .. autoclass:: psyclone.transformations.Dynamo0p3RedundantComputationTrans
     :members:
     :noindex:
+
```

Comment:
Since we're here, please could you remove the last sentence of the paragraph at L3733.

Reply:
Do you mean from the RedundantComputationTrans docstring?

Reply:
I've re-written the words to make things clearer (hopefully).

Reply:
That's above and beyond. Thanks.